firecares/settings/base.py

```diff
@@ -322,6 +322,7 @@
     Queue('quality-control', routing_key='quality-control'),
     Queue('slack', routing_key='slack'),
     Queue('weather-task', routing_key='weather-task'),
+    Queue('servicearea', routing_key='servicearea'),
```

Comment:
We'll need to have the `servicearea` queue in the VM creation for this, see https://github.com/FireCARES/firecares-ansible/issues/11